Views/Home/Index.cshtml

```diff
@@ -3,3 +3,21 @@
     <div class='separator'></div>
     <img class='logo-img' src='~/images/tcg_logo.png' alt='Logo'/>
 </div>
+
+<div class='features'>
+    <div class='feature'>
+        <img class='img' src='~/images/ic_list.png' alt='ic_list'/>
```

Comment:
I'd rather use a class name that says what's the element meaning instead of it's tag :stuck_out_tongue_closed_eyes: 
Something like `feature-image` or `feature-logo`, stuff like that